src/TableEditColumn.js

```diff
@@ -116,7 +126,12 @@ TableEditColumn.propTypes = {
   blurToSave: PropTypes.bool,
   editable: PropTypes.oneOfType([ PropTypes.bool, PropTypes.object ]),
   format: PropTypes.oneOfType([ PropTypes.bool, PropTypes.func ]),
-  children: PropTypes.node
+  fieldValue: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.bool,
+    PropTypes.number,
+    PropTypes.object
+  ])
```

Comment:
@AllenFang I'm not sure what the impact is of changing the fieldValue (which used to come in as children) from node to the 4 types I have listed.